starters/adapters/aws/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "description": "AWS Lambda",
+  "scripts": {
+    "build.server": "vite build -c adapters/aws-lambda/vite.config.ts",
+    "serverless:preview": "serverless offline"
```

Comment:
imo a deploy and remove cmd would be nice as well 👼 but would require a local aws profile

Reply:
I did not add the serverless:deploy because of that point is necessary AWS profile. I think maybe explain on the docs